db/migrate/20240713155011_update_patients_names.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class UpdatePatientsNames < ActiveRecord::Migration[7.0]
+  def change
+    patients = Patient.all
+
+    patients.each do |patient|
+      patient.update!(name: patient.name)
+    end; nil
```

Comment:
Do you need the `nil`?

Reply:
Actually I don't need it. I ran the migrate and the records of patients weren't displayed. I removed it.